src/service/bus.ts

```diff
@@ -0,0 +1,72 @@
+import buses from "../models/bus";
+
+interface BusData {
+    busNumber: string;
+    totalSeat: number;
+    availableSeat: number;
+    bookedSeat: number;
+    inAC: boolean;
+    arrival: string;
+    departure: string;
+    stoppings: string[];
+    arriveTime: string;
+    departureTime: string;
+    date: string;
+}
+
+const createBus = async (busdata: BusData): Promise<any> => {
+    const {
+        busNumber,
+        totalSeat,
+        availableSeat,
+        bookedSeat,
+        inAC,
+        arrival,
+        departure,
+        stoppings,
+        arriveTime,
+        departureTime,
+        date
+    } = busdata;
+
```

Comment:
check if the busNumber is already present and use it to handle error to avoid duplicate data

Reply:
I have created the condition controller/bus so there won't be duplicated